core-tests/src/test/java/org/mule/runtime/core/el/DataWeaveExpressionLanguageAdaptorTestCase.java

```diff
@@ -16,11 +16,7 @@
 import static org.hamcrest.Matchers.nullValue;
 import static org.hamcrest.Matchers.sameInstance;
 import static org.junit.Assert.assertThat;
-import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
No star imports. You can configure this for IntelliJ in Preferences->Editor->Code Style->Java->Imports then putting a higher number for "Class/Name count to use import with '*'".